packages/viewer/src/components/viewer-transform-widget/dom.ts

```diff
@@ -1,3 +1,6 @@
 export function focusInputElement(element: HTMLInputElement): void {
-  element.focus();
+  if (window.document.activeElement !== element) {
+    element.focus();
+    element.setSelectionRange(element.value.length, element.value.length);
+  }
```

Comment:
This change causes the input to only focus if it's not currently focused (fixes the cursor moving around when pressing the up arrow), as well as fixes an issue where the cursor would get placed at the start of the text